annotation2BIO.py

```diff
@@ -129,6 +129,8 @@ def generate_BIO(sents, entities, file_id="", no_overlap=False, record_pos=False
     for i, sent in enumerate(sents):
         nsent = []
         for j, token in enumerate(sent):
+            if (token[1][0] == token[1][1]) and (token[2][0] == token[2][1]): # Check for ZWSP characters
```

Comment:
Should we use `and` or `or` here in the if clause?
My understanding `token[1][0] == token[1][1]` check if original start and end are same; `token[2][0] == token[2][1]` check if after process start and end are same, I am not sure if we need to check both as True to skip

Reply:
Also, please add a log warning here

Reply:
Thank you for your feedback Alex. I have modified the logic to skip and added a log warning.